eduNEXT/drydock
# feat: add 1st version of rendered jobs

### Description
This PR adds a list of jobs for the most used services. This list can be configured using a variable defined in the config.yml with optional services such as `minio` or `forum`; required services like `LMS` are not removable. There's also a specified order in which they're run:
![image](https://user-images.githubusercontent.com/64440265/191133136-51920649-4118-4c2e-aa18-fb919e100855.png)
We configured this behavior using `waves` from argoCD.

### How to test
1. Specify `DRYDOCK_INIT_JOBS` with the list of services
2. Run` tutor drydock save -r reference.yml`
3. Check the rendered jobs
